GVFS/GVFS.Service/Win/WindowsRequestHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿using GVFS.Common.NamedPipes;
+using GVFS.Common.Tracing;
+using System.Runtime.Serialization;
+
+namespace GVFS.Service.Handlers
+{
+    public class WindowsRequestHandler : RequestHandler
+    {
+        public WindowsRequestHandler(
+            ITracer tracer,
+            string etwArea,
+            RepoRegistry repoRegistry) : base(tracer, etwArea, repoRegistry)
+        {
+        }
+
+        protected override void HandleMessage(
+            ITracer tracer,
+            NamedPipeMessages.Message message,
+            NamedPipeServer.Connection connection,
+            out string deserializeError)
+        {
+            if (message.Header == NamedPipeMessages.EnableAndAttachProjFSRequest.Header)
+            {
+                deserializeError = "Could not deserialize attach volume request";

```

Comment:
Why are we setting the deserializeError here? It looks like we set it to null a couple lines below...

Reply:
agree. `deserializeError` doesn't make much sense. Removed it. Added a new `requestDescriptor` field, which is then used in logging request info (in case of `SerializationException`).